alert-policies/dbt-cloud/dbt-test-warning.yml

```diff
@@ -0,0 +1,75 @@
+# Name of the alert
+name: Dbt Test Warning 
+
+# Description and details
+description: |+
```

Comment:
This description should be less than 240 characters

Reply:
Upudated